readme.adoc

```diff
@@ -23,7 +35,34 @@ A Native iOS Application that demonstrates features of the Aerogear iOS SDK.
 - Authenticated Calls to Protected Endpoints (via Keycloak)
 - Push notifications with UPS server
 
-== Work with Self-signed Certificate
+== Build Instructions
+
+=== Installing Dependencies
+To install all pods, run the following command:
+
+```
+pod install
```

Comment:
can we add a pre-req section and list the SDKs/IDEs the user need to install before building the app.

Reply:
done